bower.json

```diff
@@ -1,17 +1,10 @@
 {
   "name": "ember-frost-core",
   "dependencies": {
-    "ember": "2.11.2",
-    "ember-cli-shims": "0.1.3",
-    "ember-inflector": "^1.3.1",
-    "ember-mocha": "0.8.11",
+    "ember": "~2.12.1",
```

Comment:
It actually does need to be here because for whatever reason we are including the template compiler https://github.com/ciena-frost/ember-frost-core/blob/master/ember-cli-build.js#L26. It looks like you were the one who added that import @juwara0